ui/base/clipboard/clipboard_android.cc

```diff
@@ -102,14 +102,6 @@ void ClipboardMap::Set(const std::string& format, const std::string& data) {
 
     Java_Clipboard_setText(env, clipboard_manager_.obj(), str.obj());
   } else if (format == kHTMLFormat) {
-    // Android's API for storing HTML content on the clipboard requires a plain-
-    // text representation to be available as well. ScopedClipboardWriter has a
-    // stable order for setting clipboard data, ensuring that plain-text data
-    // is available first. Do not write to the clipboard when only HTML data is
-    // available, because otherwise others apps may not be able to paste it.
-    if (!ContainsKey(map_, kPlainTextFormat))
-      return;
-
```

Comment:
Please refer to :https://code.google.com/p/chromium/issues/detail?id=469565

Currently, copy/paste picture only is intentionally disabled. Because of copy/paste picture only isn't being good supported in Android system. The detail behavior of current chrome for android are listed below:
1),Copy picture(s) with text from chrome for Android and pastes it into an Android 
apps, it seldom works,picture(s) turns into "OBJ".
2),Copy picture(s) only from chrome for Android and pastes it into an Android apps, it seldom works,picture(s) turns into "OBJ".
3),Copy/paste picture(s) with text in chrome for Android, it works.
4),Copy/paste picture(s) only in chrome for Android, it doesn't work.

It's poor user experience for item 3) and item 4). 
So this fix is intent to  align item 3) and 4).